storage/disk/disk.go

```diff
@@ -348,6 +348,19 @@ func (db *Store) Truncate(ctx context.Context, txn storage.Transaction, params s
 
 	db.rmu.Lock()
 
+	// update symlink to point to the active db
+	symlink := filepath.Join(path.Dir(newDB.Opts().Dir), symlinkKey)
+	if _, err := os.Lstat(symlink); err == nil {
```

Comment:
Let's play it safe and check the error here. Some sort of "not found" error is what we expect, I think?

Reply:
Updated this path to add some error handling. LMKWYT